services/github-service-api/src/main/java/org/kontinuity/catapult/service/github/api/GitHubRepository.java

```diff
@@ -12,7 +13,7 @@
     /**
      * @return the "git" protocol transport URL
      */
-    String getGitTransportUrl();
+    URI getGitTransportUrl();
```

Comment:
get*Url but returns URI?

Let's pick one and stick with it. :)

Also TBH I think we can just kill this method safely; only some extraneous log statement is using it.